src/auro-nav.js

```diff
@@ -292,14 +311,14 @@ export class AuroNav extends LitElement {
         <slot @slotchange="${this.handleSlotItems}"></slot>
       </div>
       ${this.anchorlinks && this.anchorlinks.length > this.mobileViewCollapsedNumLinks ? html`
-        <auro-button slim tertiary class="showHideBtn" @click=${this.toggleAnchorLinks}>
+        <${this.buttonTag} slim tertiary class="showHideBtn" @click=${this.toggleAnchorLinks} id="anchorlinkButton">
```

Comment:
It looks like the ID is only added to make a test pass. Can't you just make the test selector look for the class `showhideBtn` that is already on the element?